### ISSUE TITLE
Google Play Protect

### ISSUE THREAD

**Issue by omnigenous** [cake-tech/cake_wallet#1599]
**Describe the bug**

This app may be harmful

**To Reproduce**

Install 4.19.1 from f-droid repo (from 4.18.2)

**Screenshots**

![image](https://github.com/user-attachments/assets/69b8d92b-aadf-4549-991a-85db34f4b9be)

**Platform:**
 - OS:  Android 9
 - Cake Wallet Version: 4.19.1 (Trying to install)

**Comment by terryschmidt** [comment]
Got the same message just now on 4.19.4. I was installing via Obtainium.

**Comment by omnigenous** [comment]
`4.18.2 -> 4.19.5` same issue

**Comment by omnigenous** [comment]
https://github.com/cake-tech/cake_wallet/compare/v4.18.2...v4.19.1

Quite a few changes... I'm unsure where to begin finding what causing this warning since I'm not an android developer.

**Comment by omnigenous** [comment]
@tuxpizza @OmarHatem28 sorry for the ping, but can this issue be addressed/acknowledged?

**Comment by OmarHatem28** [comment]
I believe this is the default behavior of google play protect, to assume any app that is installed from an external source other than google play, as harmful, so I believe this is the expected behavior

**Comment by omnigenous** [comment]
Wasn't an issue in 4.18.2 or below, do you know why?

**Comment by OmarHatem28** [comment]
nope, not sure why, maybe google updated how they check for harmful apps, and since CakeWallet has Monero code which is treated as malware by multiple entities, then maybe it got flagged

**Comment by terryschmidt** [comment]
> I believe this is the default behavior of google play protect, to assume any app that is installed from an external source other than google play, as harmful, so I believe this is the expected behavior

I don't think it's true. I have 8 open source apps that I install and keep updated through Obtainium on my Pixel 9 Pro, and only Cake is being reported as harmful by Play Protect.

**Comment by tuxpizza** [comment]
Does it show this everytime you update Cake? For instance 4.20 -> 4.20.1

**Comment by omnigenous** [comment]
Well I'm still on `4.18.2` because of this issue.

**Comment by tuxpizza** [comment]
You can just tap "Install anyway"

**Comment by automainint** [comment]
I can't install the update because of this. Turning off Play Protect or "Install anyway" doesn't work. I click it, it asks for the pin code, and then nothing happens; Play Protect is still on, the app blocking notification just shows up repeatedly. The phone is Redmi 4X.

**Comment by proofy** [comment]
I had the same problem with the Taler Wallet (https://git.taler.net/taler-android.git/) . They have fixed it, but I don't know how.

**Comment by pannal** [comment]
Still an issue, unfortunately.

**Comment by MrCyjaneK** [comment]
You will get that warning when installing pretty much any other app outside of play store (afaik even Flutter hello world app will trigger this, and mind you there's the same company behind both), not to mention the fact that we do ship monero's randomx algorithm which is flagged by pretty much anything and not to mention the fact that according to google this is a counterfeit Cake Wallet because it doesn't match the one on Play Store - due to the way in which play store signing works.

If you do believe in Play Protect and use it as measure of security I'd recommend installing official distribution of Cake Wallet directly from Play Store.

**Comment by pannal** [comment]
Hmm, I wish that were true. I have ~25 non-play-store apps that regularly update and are never flagged. So this is explicit to Cake Wallet in my case. That Protect detects monero patterns is a sound explanation, though.

**Comment by MrCyjaneK** [comment]
Are the non-play-store apps distributed using different builds for Play Store and whatever other source you use while still having the same package name?
